feat: Add more storage options to Lens Client

jonathangus opened the pull request:
<!-- Use this template by filling in information and copy and pasting relevant items out of the html comments. -->

<!-- LINK TO ISSUE OR TICKET -->
<!-- This risks section is to be filled out before final review and merge. -->

# Risks

Low

# Background

## What does this PR do?
Enable more storage options in Lens Client. With this PR we now support:
- Arweave
- Storj (default, to not introduce any breaking changes)
- Pinata

---
Fix: Improve`getProfilePictureUri` to use correct types instead of `any`

## What kind of change is this?

Improvements


<!-- This "Why" section is most relevant if there is no linked issue explaining why. If there is a related issue it might make sense to skip this why section. -->

## Why are we doing this? Any context or related work?
Wanted to use Lens Client and posts where stored on Storj that required a beta account so wanted to introduce more options.

Fix TODO for `packages/client-lens/src/actions.ts` in https://github.com/elizaOS/eliza/issues/1223


# Documentation changes needed?

<!--
My changes do not require a change to the project documentation.
My changes require a change to the project documentation.
If a docs change is needed: I have updated the documentation accordingly.
-->

<!-- Please show how you tested the PR. This will really help if the PR needs to be retested, and probably help the PR get merged quicker. -->

# Testing

## Where should a reviewer start?
Create an agent with Lens Client enabled

## Detailed testing steps
Change the environment variables for each storage provider

**Storj**: 
```
LENS_STORAGE_PROVIDER=storj
STORJ_API_USERNAME=
STORJ_API_PASSWORD=
```

**Arweave**: 
```
LENS_STORAGE_PROVIDER=arweave
ARWEAVE_JWK=
```

**Pinata**:
```
LENS_STORAGE_PROVIDER=pinata
PINATA_JWT=
PINATA_GATEWAY_URL=
```


<!--
None, automated tests are fine.
-->

<!--
- As [anon/admin], go to [link]
  - [do action]
  - verify [result]
-->

<!-- If there is a UI change, please include before and after screenshots or videos. This will speed up PRs being merged. It is extra nice to annotate screenshots with arrows or boxes pointing out the differences. -->
<!--
## Screenshots
### Before
### After
-->

<!-- If there is anything about the deploy, please make a note. -->
<!--
# Deploy Notes
-->

<!--  Copy and paste commandline output. -->
<!--
## Database changes
-->

<!--  If there is something more than the automated steps, please specifiy deploy instructions. -->
<!--
## Deployment instructions
-->

<!-- If you are on Discord, please join https://discord.gg/ai16z and state your Discord username here for contribute role and join us in #development-feed -->

## Discord username
@0xheavydev


<!-- This is an auto-generated comment: release notes by coderabbit.ai -->

## Summary by CodeRabbit

- **Configuration**
	- Added new environment variables for Lens Protocol integration
	- Introduced configuration options for storage providers (Arweave, Pinata, Storj)

- **New Features**
	- Enhanced storage provider flexibility
	- Added support for multiple storage backends
	- Improved Lens Protocol interaction management

- **Dependencies**
	- Added Arweave library (version 1.15.5)

- **Improvements**
	- Refactored storage provider interfaces
	- Updated profile picture URI handling
	- Improved error logging and configuration management

<!-- end of auto-generated comment: release notes by coderabbit.ai -->

jonathangus commented:
@imthatcarlos wdyt?

imthatcarlos commented:
good stuff @jonathangus - i was hoping someone would take care of this 😅

imthatcarlos commented:
@jonathangus I've looked thru the storj provider changes and it should be fine. Plus, I think the vast majority of Lens eco uses Arweave anyways. 

LGTM

jonathangus commented on .env.example:
Not used anymore

jonathangus reviewed (commented):
Resolved conflicts in .env.example and removed 2 unused envs

AIFlowML committed:
Merge branch 'main' into fix-plugin-agentkit

AIFlowML committed:
fixed all the issues but deployTokenAction.ts for the ANY and the result need probaly a refactoring.

AIFlowML committed:
Merge branch 'develop' into fix-plugin-abstract

AIFlowML committed:
Merge branch 'develop' into fix-plugin-agentkit

0xCardiE committed:
add new pools by network

AIFlowML committed:
Fixed all the issues and added biome

AIFlowML committed:
Merge branch 'develop' into fix-plugin-3g-gen

AIFlowML committed:
Fixed all the issues and added Biome

AIFlowML committed:
Merge branch 'develop' into fix-plugin-0x

0xCardiE committed:
add mappings from provider

AIFlowML committed:
Fixed all the issues and added biome. Only issues still in place in upload.ts i had to add an any to overcome the issue on the line 370 const flowContract = getFlowContract(runtime.getSetting(ZEROG_FLOW_ADDRESS), signer as any);

AIFlowML committed:
Merge branch 'develop' into fix-plugin-0g

AIFlowML committed:
Added biome to - plugin-zksync-era - plugin-zilliqa - plugin-zerion - plugin-whatsapp

AIFlowML committed:
Merge branch 'develop' into fix-multi-biome-01

AIFlowML committed:
Add Biome linting and formatting configuration to the following plugins:
- plugin-web-search
- plugin-video-generation
- plugin-udio
- plugin-twitter

shakkernerd committed:
Merge pull request #3180 from AIFlowML/fix-multi-biome-01

fix: multi-biome-01

shakkernerd committed:
Merge branch 'develop' into fix-plugin-0g

AIFlowML committed:
Merge branch 'develop' into fix-multi-biome-02

shakkernerd committed:
Merge pull request #3179 from AIFlowML/fix-plugin-0g

fix: plugin-0g

shakkernerd committed:
Merge branch 'develop' into fix-multi-biome-02

shakkernerd committed:
Merge pull request #3181 from AIFlowML/fix-multi-biome-02

fix: multi-biome-02

shakkernerd committed:
chore: remove multiple tsup package

shakkernerd committed:
Merge branch 'develop' into fix-plugin-0x

shakkernerd committed:
Merge pull request #3178 from AIFlowML/fix-plugin-0x

fix: plugin-0x

shakkernerd committed:
Merge branch 'develop' into fix-plugin-3g-gen

shakkernerd committed:
Merge pull request #3175 from AIFlowML/fix-plugin-3g-gen

fix: plugin-3g-generation

shakkernerd committed:
Merge branch 'develop' into fix-plugin-abstract

shakkernerd committed:
Merge pull request #3174 from AIFlowML/fix-plugin-abstract

fix: plugin-abstract

shakkernerd committed:
Merge branch 'develop' into fix-plugin-agentkit

shakkernerd committed:
Merge pull request #3172 from AIFlowML/fix-plugin-agentkit

fix: plugin-agentkit

shakkernerd committed:
Merge branch 'develop' into fix-plugin-akash-v1

shakkernerd committed:
Merge pull request #3171 from AIFlowML/fix-plugin-akash-v1

fix: plugin-akash

shakkernerd committed:
Merge branch 'develop' into fix-plugin-allors

shakkernerd committed:
Merge pull request #3169 from AIFlowML/fix-plugin-allors

fix: plugin-allora

shakkernerd committed:
Merge branch 'develop' into main

shakkernerd committed:
Merge pull request #3165 from comfsrt/main

docs: fix typos in .md

AIFlowML committed:
Add Biome linting and formatting configuration to the following plugins:
- plugin-ton
- plugin-trikon
- plugin-trustdb
- plugin-tts

Changes include:
- Add @biomejs/biome as devDependency
- Add lint and format scripts
- Replace existing ESLint configurations with Biome
- Maintain existing build and test scripts

ai16z-demirix committed:
feat (chore): plugin-coinmarketcap (#3134)

* plugin-coinmarketcap: vitest config

* plugin-coinmarketcap: package json

* plugin-coinmarketcap: test setup

* plugin-coinmarketcap: get price tests

* plugin-coinmarketcap: get price service tests

---------

Co-authored-by: Shakker Nerd <165377636+shakkernerd@users.noreply.github.com>

ai16z-demirix committed:
feat (chore): plugin-coingecko test config and coverage (#3124)

* plugin-coingecko: package json

* plugin-coingecko: vitest config

* plugin-coingecko: get price tests

* plugin-coingecko: test setup

* plugin-coingecko: get trending tests

* plugin-coingecko: top gainers losers tests

* plugin-coingecko: get market tests

---------

Co-authored-by: Shakker Nerd <165377636+shakkernerd@users.noreply.github.com>

AIFlowML committed:
Add Biome linting and formatting configuration to the following plugins:
- plugin-thirdweb
- plugin-tee-verifiable-log
- plugin-tee-marlin
- plugin-tee-log

AIFlowML committed:
Add Biome linting and formatting configuration to the following plugins:
- plugin-tee
- plugin-suno
- plugin-sui
- plugin-story

AIFlowML committed:
Add Biome linting and formatting configuration to the following plugins:
- plugin-starknet
- plugin-stargaze
- plugin-squid-router
- plugin-spheron

AIFlowML committed:
Add Biome linting and formatting configuration to the following plugins:
- plugin-solana-v2
- plugin-solana-agent-kit
- plugin-solana
- plugin-sgx

AIFlowML committed:
Merge branch 'develop' into fix-multi-biome-07

tcm390 committed:
Merge branch 'main' into tcm/remove-duplicated-plugins

tcm390 committed:
Merge branch 'develop' into tcm/remove-duplicated-plugins

tcm390 committed:
Merge pull request #3126 from elizaOS/tcm/remove-duplicated-plugins

fix: remove duplicated plugins

tcm390 committed:
update provider-utils

tcm390 committed:
Merge branch 'develop' into tcm-update-provider-utils

tcm390 committed:
Merge pull request #3189 from elizaOS/tcm-update-provider-utils

fix: update provider-utils

tcm390 committed:
extract attribute from raw text instead of normalized json

tcm390 committed:
Merge pull request #3190 from elizaOS/tcm-correct-extract

fix: extract attribute from raw text instead of normalized json

samarth30 committed:
Merge branch 'develop' into fix-multi-biome-07

samarth30 committed:
Merge branch 'develop' into feat/coingecko_advanced_more_endpoints

john-xina-p88 committed:
Merge branch 'develop' into develop

samarth30 committed:
Merge pull request #3170 from 0xCardinalError/feat/coingecko_advanced_more_endpoints

feat: coingecko advanced -  various pools by network

samarth30 committed:
Merge pull request #3045 from edwin-finance/plugin-edwin

feat: Add edwin plugin to eliza

samarth30 committed:
Merge pull request #3096 from HMXOrg/develop

feat: plugin desk exchange

maxime committed:
fix: make slack client able to download and upload attachments

nilaysarma committed:
Update Twitter to X (Twitter)

Since Twitter is now rebranded to X, so I have changes made changes in the README file. It is now "X (Twitter)" and I kept the Twitter keyword since people still use the word Twitter.

tcm390 committed:
Merge branch 'develop' into 3196--Update-Twitter-to-X

tcm390 committed:
Merge pull request #3198 from nilaysarma/3196--Update-Twitter-to-X

docs: Update Twitter to X (Twitter)

tcm390 committed:
Merge branch 'develop' into fix/slack_download_upload_attachments

Marcofann committed:
Update index.ts

shakkernerd committed:
Merge branch 'develop' into fix-multi-biome-07

shakkernerd committed:
Merge pull request #3186 from AIFlowML/fix-multi-biome-07

chore: add Biome configuration to Solana ecosystem plugins 07

gap-editor committed:
typo chat_2024-12-01.md

solono - solana

gap-editor committed:
typo chat_2024-12-03.md

vides - videos

gap-editor committed:
Update chat_2024-10-30.md

Docker composes - Docker Compose

shakkernerd committed:
Merge branch 'develop' into fix-plugin-anyone

shakkernerd committed:
Merge pull request #3107 from AIFlowML/fix-plugin-anyone

fix: plugin-anyone

YoungPhlo committed:
Merge branch 'develop' into docs/contributor-meetings

rferrari committed:
fix: think tag from venice

YoungPhlo committed:
docs: Add weekly contributor meeting notes (01-21 and 01-28)

gap-editor committed:
Update chat_2024-11-23.md

gap-editor committed:
typo chat_2024-12-02.md

YoungPhlo committed:
capitalized ElizaOS (for graphite)

Danyylka committed:
Update faq.md

shakkernerd committed:
Merge branch 'develop' into main

shakkernerd committed:
Merge pull request #3207 from Danyylka/main

docs: Update faq.md

shakkernerd committed:
Merge branch 'develop' into main

shakkernerd committed:
Merge pull request #3206 from gap-editor/main

chore: Fix Typos and Improve Consistency in Community Chat Logs

shakkernerd committed:
Merge branch 'develop' into docs/contributor-meetings

shakkernerd committed:
Merge pull request #3204 from YoungPhlo/docs/contributor-meetings

docs: Add weekly contributor meeting notes

shakkernerd committed:
Merge branch 'develop' into develop

shakkernerd committed:
Merge pull request #3203 from rferrari/develop

fix: think tag from venice

shakkernerd committed:
Merge branch 'develop' into fix/slack_download_upload_attachments

shakkernerd committed:
Merge pull request #3194 from maxime/fix/slack_download_upload_attachments

fix: Slack download upload attachments

shakkernerd committed:
Merge branch 'develop' into main

shakkernerd committed:
Merge pull request #3166 from PixelPil0t1/main

chore: Update GitHub Actions workflows and documentation

shakkernerd committed:
Merge branch 'develop' into fix/fix

shakkernerd committed:
Merge pull request #3164 from Marcofann/fix/fix

chore: Standardization of Security Check Identifiers across GoPlus plugin

shakkernerd committed:
Merge branch 'develop' into docker-pnpm-version

shakkernerd committed:
Merge pull request #3158 from v1xingyue/docker-pnpm-version

fix: Update pnpm version during Docker build

shakkernerd committed:
Merge branch 'develop' into fix-readme-ja

shakkernerd committed:
Merge pull request #3153 from You-saku/fix-readme-ja

Fix README_JA.md (add unwritten text and fix typo)

shakkernerd committed:
Merge branch 'develop' into some-fix

shakkernerd committed:
chore: conditional logging for server starting on a different port

shakkernerd committed:
Merge pull request #3141 from v1xingyue/some-fix

chore: some fix after v0.1.9

silasneo committed:
feat: Trump character but tweets in Spanish by default (#3119)

* Update README.md

small note at the top of readme to highlight contribution via this fork

* custom character file

* Trump in Latin American Spanish

Modified Trump character file to have the agent speak in Latin Spanish by default.
The agent will respond in any language if specifically asked to respond in that language in the user input.
This file can serve as a template for creating agents who speak a specific language by default for a given language audience.

* Update README.md

* Delete characters/uncleugo4.character.json

* Adding spanish_trump.character.json

Many people are not native English speakers & a template was needed to make it easier for anyone to create an agent that speaks other languages by default. 
Modified Trump character file to have the agent speak in Latin Spanish by default. 
This file can serve as a template for creating agents who speak a specific language by default for a given language audience.
The agent will respond in any language if specifically asked to respond in that language in the user input.

* Update spanish_trump.character.json

minor correction for duplicated adjectives

* Update spanish_trump.character.json

* removed duplicate adjectives

* Update package.json

* Update spanish_trump.character.json

Removed some English in bio for consistency

---------

Co-authored-by: Shakker Nerd <165377636+shakkernerd@users.noreply.github.com>

azep-ninja committed:
optimizations & fixes.

elpulpo0 committed:
feat: Add CREATE_POOL action in MultiversX plugin

Fallengirl committed:
Update chat_2024-11-17.md

elpulpo0 committed:
feat: update constants in MultiversX plugin

mikirov committed:
feat: nft actions

odilitime committed:
Merge pull request #3211 from mikirov/feat/ton-plugin-nft-management

feat: TON Plugin: NFT collection, item creation, metadata change and transfer actions

odilitime committed:
Merge pull request #3210 from Fallengirl/patch-1

docs: rename chat_2024-11-17.md

odilitime committed:
Merge pull request #3209 from elpulpo0/develop

feat: Add CREATE_POOL action in MultiversX plugin

odilitime committed:
Merge branch 'develop' into fix-quickintel-plugin

tcm390 committed:
remove duplicated dependencies

tcm390 committed:
Merge pull request #3215 from elizaOS/tcm-remove-duplicated-dependencies

fix: remove duplicated dependencies

shakkernerd committed:
Merge branch 'develop' into fix-quickintel-plugin

shakkernerd committed:
Merge pull request #3208 from azep-ninja/fix-quickintel-plugin

fix: quick-intel plugin optimizations & fixes.

tcm390 committed:
add ENABLE_TWITTER_POST_GENERATION config

tcm390 committed:
add ENABLE_TWITTER_POST_GENERATION config

tcm390 committed:
add ENABLE_TWITTER_POST_GENERATION

shakkernerd committed:
Merge pull request #3219 from elizaOS/tcm-twitter-disable-post

feat: Add configuration for enabling/disabling Twitter post generation

odilitime committed:
add zod

odilitime committed:
bump zod

odilitime committed:
add new dependency

odilitime committed:
bump lock

odilitime committed:
Merge branch 'develop' into odi-fix

elpulpo0 committed:
feat: (plugin-multiversx) Allow the use of herotag instead of address to send token

cypherpepe committed:
Update broken link awesome-eliza.md

cypherpepe committed:
Update broken link README.md

cypherpepe committed:
Update broken link README.md

cypherpepe committed:
Update broken link profiles.mdx

cypherpepe committed:
Update broken link chat_2024-11-16.md

ai16z-demirix committed:
plugin-coflux: package json

ai16z-demirix committed:
plugin-coflux: vitest config

ai16z-demirix committed:
plugin-coflux: setup

ai16z-demirix committed:
plugin-coflux: bridge transfer tests

ai16z-demirix committed:
plugin-coflux: transfer tests

azep-ninja committed:
rag optimizations/fixes for context.

ai16z-demirix committed:
plugin-cronos: vitest config

ai16z-demirix committed:
plugin-cronos: package json

ai16z-demirix committed:
plugin-cronos: setup

ai16z-demirix committed:
plugin-cronos: balance tests

ai16z-demirix committed:
plugin-cronos: transfer tests

odilitime committed:
Merge branch 'develop' of https://github.com/elizaos/eliza into odi-fix

madschristensen99 committed:
Delete packages/plugin-lit/src/providers directory

madschristensen99 committed:
Delete packages/plugin-lit/src/config directory

madschristensen99 committed:
Delete packages/plugin-lit/src/actions/sendSol.ts

madschristensen99 committed:
Delete packages/plugin-lit/src/actions/sendEth.ts

madschristensen99 committed:
Delete packages/plugin-lit/src/actions/sendUSDC.ts

madschristensen99 committed:
Delete packages/plugin-lit/src/actions/index.ts

madschristensen99 committed:
Delete packages/plugin-lit/src/types directory

madschristensen99 committed:
Delete packages/plugin-lit/pkp-setup.md

madschristensen99 committed:
Delete packages/plugin-lit/.npmignore

madschristensen99 committed:
Update README.md

shakkernerd committed:
Merge pull request #3250 from ai16z-demirix/tests/plugin-cronos

feat (chore): plugin-cronos test setup and coverage

shakkernerd committed:
Merge branch 'develop' into tests/plugin-conflux

shakkernerd committed:
Merge pull request #3247 from ai16z-demirix/tests/plugin-conflux

feat (chore) plugin conflux: test config and coverage

shakkernerd committed:
Merge branch 'develop' into main

shakkernerd committed:
Merge pull request #3240 from cypherpepe/main

fix: broken links in documentation

AIFlowML committed:
Add Biome linting and formatting configuration to client-alexa:
- Add @biomejs/biome as devDependency
- Add lint and format scripts
- Replace existing ESLint configuration
- Standardize tsup dependency version

shakkernerd committed:
Merge branch 'develop' into main

shakkernerd committed:
Merge pull request #3251 from madschristensen99/main

chore: remove unnecessary provider and transfer code

shakkernerd committed:
Merge branch 'develop' into fix-client-alexa

shakkernerd committed:
Merge pull request #3255 from AIFlowML/fix-client-alexa

fix: client-alexa

renovate[bot] committed:
fix(deps): update dependency vitest [security] (#3254)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: Shakker Nerd <165377636+shakkernerd@users.noreply.github.com>

azep-ninja committed:
Merge branch 'develop' into fix-rag-optimizations

tcm390 committed:
handle invalid json format

tcm390 committed:
trim input

tcm390 committed:
use raw text to normalized

tcm390 committed:
Merge branch 'develop' into tcm-handle-invalid-json

tcm390 committed:
fix install

tcm390 committed:
Merge pull request #3261 from elizaOS/tcm-fix-install

fix: pnpm install

tcm390 committed:
fix http dynamic require issue

tcm390 committed:
Merge branch 'develop' into tcm/fix-plugin-ton

tcm390 committed:
Merge pull request #3262 from elizaOS/tcm/fix-plugin-ton

fix: plugin-ton Dynamic require of "http" is not supported

esen committed:
Fix PG query: Use only 'text' subfield name, field name is not needed here

nilaysarma committed:
add github issues link

johntad110 committed:
Fix broken links on contributing.md

rebustron committed:
Update broken link README.md

lalalune committed:
fix develop build issue

lalalune committed:
Merge branch 'develop' of http://github.com/ai16z/eliza into HEAD

lalalune committed:
Add chracters

lalalune committed:
remove minified code block

lalalune committed:
Merge pull request #3269 from johntad110/fix-broken-links

Fix broken links on contributing.md

lalalune committed:
Merge pull request #3268 from nilaysarma/add-link-to-issues

docs: Add GitHub issues link to CONTRIBUTING.md file

lalalune committed:
Merge pull request #3264 from esen/fix_pg_memory_query

Fix PG query: Use only 'text' subfield name, field name is not needed here

lalalune committed:
Merge branch 'develop' into tcm-handle-invalid-json

lalalune committed:
Merge pull request #3258 from elizaOS/tcm-handle-invalid-json

fix: handle invalid json

rebustron committed:
typo corr README.md

lalalune committed:
Merge branch 'develop' of http://github.com/ai16z/eliza into HEAD

rebustron committed:
typo corr 2025-01-17.md

reject-i committed:
Update chat_2024-11-19.md

reject-i committed:
Update chat_2024-11-22.md

reject-i committed:
Update chat_2024-06-20.md

reject-i committed:
Update README.md

tcm390 committed:
add support for Twitter cookies from environment settings

tcm390 committed:
add support for Twitter cookies from environment settings

shakkernerd committed:
Merge branch 'develop' into main

shakkernerd committed:
Merge pull request #3270 from rebustron/main

docs: fix typos and update broken link in docs

shakkernerd committed:
Merge branch 'develop' into main

shakkernerd committed:
Merge pull request #3271 from XxAlex74xX/main

chore: fix spelling issues

btspoony committed:
fix(core): improve OpenAI-like provider endpoint resolution in `generation.ts`

tcm390 committed:
rename variable

tcm390 committed:
Merge branch 'develop' into tcm-fix-DenyLoginSubtask

shakkernerd committed:
chore: add description text to env example

shakkernerd committed:
Merge pull request #3278 from elizaOS/tcm-fix-DenyLoginSubtask

fix: DenyLoginSubtask

azep-ninja committed:
Merge branch 'develop' into fix-rag-optimizations

azep-ninja committed:
optimize template/add suppress init msg

azep-ninja committed:
add actions suppress action ability.

azep-ninja committed:
add actions suppress action ability.

azep-ninja committed:
add actions suppress action ability.

shakkernerd committed:
Merge pull request #3286 from azep-ninja/fix-twitter-suppress-message-add

fix: twitter - add actions suppress action ability.

shakkernerd committed:
Merge branch 'develop' into fix-rag-optimizations

shakkernerd committed:
Merge pull request #3248 from azep-ninja/fix-rag-optimizations

fix: rag optimizations/fixes for context.

shakkernerd committed:
Merge branch 'develop' into fix-optimize-quick-intel-plugin

shakkernerd committed:
Merge pull request #3283 from azep-ninja/fix-optimize-quick-intel-plugin

fix: quick-intel plugin: optimize template/add suppress init msg

shakkernerd committed:
Merge branch 'develop' into fix-discord-suppress-message-add

shakkernerd committed:
Merge pull request #3284 from azep-ninja/fix-discord-suppress-message-add

fix: discord - add actions suppress action ability.

shakkernerd committed:
Merge branch 'develop' into fix-telegram-suppress-message-add

shakkernerd committed:
Merge pull request #3285 from azep-ninja/fix-telegram-suppress-message-add

fix: telegram- add actions suppress action ability.

shakkernerd committed:
Merge branch 'develop' into develop

shakkernerd committed:
Merge pull request #3238 from elpulpo0/develop

feat: (plugin-multiversx) Allow the use of herotag

odilitime committed:
Merge branch 'develop' of https://github.com/elizaos/eliza into odi-fix

odilitime committed:
bump lock

odilitime committed:
Merge branch 'odi-fix' of https://github.com/elizaos/eliza into odi-fix

odilitime committed:
add zod globally

odilitime committed:
remove zod

odilitime committed:
bump lockfile

tcm390 committed:
clean json before normalized

odilitime committed:
Merge pull request #3220 from elizaOS/odi-fix

fix: fix docker & types issue

odilitime committed:
bump version to 0.25.6-alpha.1

odilitime committed:
bump lock

tcm390 committed:
Merge branch 'develop' into tcm-clean-json

tcm390 committed:
Merge branch 'develop' into tbh/fix-openai-like-provider-endpoint

odilitime committed:
Merge branch 'develop' of https://github.com/elizaos/eliza into odi-fix

tcm390 committed:
Merge pull request #3281 from btspoony/tbh/fix-openai-like-provider-endpoint

fix(core): improve OpenAI-like provider endpoint resolution in `generation.ts`

tcm390 committed:
Merge branch 'develop' into tcm-clean-json

odilitime committed:
Merge branch 'develop' into odi-fix

odilitime committed:
Merge pull request #3306 from elizaOS/odi-fix

chore: bump version to 0.25.6-alpha.1

tcm390 committed:
Merge branch 'develop' into tcm-clean-json

odilitime committed:
Merge pull request #3301 from elizaOS/tcm-clean-json

fix: clean json before normalized

oxy-Op committed:
Fix plugin-solana-v2 package.json (#3308)

Expected double-quoted property name in JSON at position 825 while parsing '{    "name": "@elizaos/plugin-solana-v2' in /home/oxy/everreach-ai-agent/packages/plugin-solana-v2/package.json

Co-authored-by: Odilitime <janesmith@airmail.cc>

tcm390 committed:
set initial value to null

tcm390 committed:
return string

tcm390 committed:
return null of empty object

tcm390 committed:
Merge pull request #3311 from elizaOS/tcm-parsing-unit-test

fix: parsing unit test failure

tcm390 committed:
use latest getModelSettings function

tcm390 committed:
fix model test

tcm390 committed:
update LIVEPEER endpoint

tcm390 committed:
Merge branch 'develop' into tcm/fix-models-unit-test

tcm390 committed:
Merge pull request #3312 from elizaOS/tcm/fix-models-unit-test

fix: models unit test

jonathangus committed:
Merge branch 'r-develop' into develop

jonathangus committed:
spelling

jonathangus committed:
Merge branch 'develop' into develop

odilitime commented:
Hello,

We are changing our plugin development strategy to be more scalable. We have moved the plugins out into their own repos and we're looking for people to either maintain those or own them on their own Github.

Please make your code changes to this repo: https://github.com/elizaos-plugins/client-lens

If you'd like to be a maintainer,  file an issue in the plugin repo and join our Discord https://discord.gg/elizaos to coordinate.

If you'd like to control the plugin on your own Github, please add an issue to the plugin repo pointing to your repo, and add a modification to the registry. Submit a PR to edit the registry here: https://github.com/elizaos-plugins/registry

Closing this PR for now. Let us know if you have any questions.
